packages/cli-lib/projects.js

```diff
@@ -71,7 +71,6 @@ async function downloadProject(
       encoding: null,
       headers: { ...DEFAULT_USER_AGENT_HEADERS },
     });
-    logger.log('Completed project fetch.');
```

Comment:
It might be worth it to keep these and switch to the debug level (`logger.debug`) to aid in troubleshooting in the future

Reply:
I can't do it because I switch level on my local machine but it prints debug messages